data-esnext.js

```diff
@@ -4144,6 +4144,34 @@ exports.tests = [
     },
   ]
 },
+{
+  name: 'Hashbang Grammar',
+  category: STAGE3,
+  significance: 'tiny',
+  spec: 'https://github.com/tc39/proposal-hashbang/',
+  mdn: 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#Hashbang_comments',
+  exec: function() {/*
+    try {
+      return !eval('#!/wash/your/hands');
+    } catch (e) {
+      return false
+    }
+  */},
+  res: {
+    chrome1: false,
+    chrome74: true,
+    firefox2: false,
+    firefox67: true,
+    ie11: false,
+    opera10_50: false,
+    edge18: false,
+    safari13: null,
```

Comment:
I don't have `safari13` environments in hands, so I will be appreciated if you can suggest `safari13` support data. The other major browser support data is copied from https://github.com/tc39/proposal-hashbang#status

Reply:
Locally on Safari@13.0.5, `safari13` returns `false`